src/codewhisperer/util/telemetryHelper.ts

```diff
@@ -340,6 +340,13 @@ export class TelemetryHelper {
             e2eLatency = performance.now() - session.invokeSuggestionStartTime
         }
 
+        let codewhispererRuntimeLanguage: string = this.sessionDecisions[0].codewhispererLanguage
```

Comment:
I think we already have a method `mapToRuntimeLanguage` can directly use that?

Reply:
In telemetry we do want to report if it's tsx or jsx specific.

Reply:
I thought we are mapping it to typescript/javascript here?

Reply:
for that function 
```
public mapToRuntimeLanguage<
        T extends codewhispererClient.ListRecommendationsRequest | codewhispererClient.GenerateRecommendationsRequest
    >(request: T): T
```


```
return {
            ...request,
            fileContext: {
                ...fileContext,
                programmingLanguage: parentLanguage,
            },
        }
```

Reply:
we only map for the request to the service but for telemetry we are stil reporting tsx if it's tsx.

Reply:
actually nvm, I thought we are changing for Kibana metrics but this is STE. So this is good then.